components/lvgl_ili9341/ili9488.c

```diff
@@ -168,7 +169,7 @@ void ili9488_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * col
 	/*Memory write*/
 	ili9488_send_cmd(ILI9488_CMD_MEMORY_WRITE);
 
-#ifdef 24BIT_DATA
+#if defined DATA_24BITS
 	ili9488_send_color((void*) buffer_24bit, size * 3);
 #else
 	ili9488_send_color((void*) buffer_16bit, size * 3);
```

Comment:
Change to:

	ili9488_send_color((void*) mybuf, size * 3);
	heap_caps_free(mybuf);

Reply:
Addressed with ef6be68, one question tho, do we need to free the allocated memory on the flush function or when the DMA engine is done transferring it?